src/app/containers/App/App.jsx

```diff
@@ -64,7 +66,7 @@ export const App = ({ routes, location, initialData, bbcOrigin }) => {
       };
       fetchData();
     }
-  }, [routes, location.pathname]);
+  }, [routes, location.pathname, dials]);
```

Comment:
Are dials going to change during a single render? You can likely pass dials directly, like `bbcOrigin` on line 71

Reply:
Ah didn't spot you could do that - thanks. Changed.